lib/msal-common/src/response/AuthenticationResult.ts

```diff
@@ -42,4 +42,5 @@ export type AuthenticationResult = {
     msGraphHost?: string;
     code?: string;
     fromNativeBroker?: boolean;
+    httpVer?: string;
```

Comment:
Is this something we should be returning or is this just for our Telemetry?

Reply:
this is just for telemetry as discussed.